doc/source/ray-air/getting-started.rst

```diff
@@ -3,37 +3,46 @@
 Ray AI Runtime (AIR)
 ====================
 
-Ray AI Runtime (AIR) is an open-source toolkit for building end-to-end ML applications. By leveraging Ray, its distributed compute capabilities, and its library ecosystem, Ray AIR brings scalability and programmability to ML platforms.
+Ray AI Runtime (AIR) is a scalable and unified toolkit for ML applications. AIR enables easy scaling of individual workloads, end-to-end workflows, and popular ecosystem frameworks. Unlike other toolkits, AIR can scale workloads with just Python by leveraging the Ray unified compute framework.
```

Comment:
If I'm a user trying to do batch scoring, I can't tell within the first page whether this is a tool for me... the copy is really high level.

Reply:
the batch or online scoring is a function of when you want to use AIR. So it should come under the subheading of When to use AIR? Just as Why Ray AIR?